proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -2282,6 +2437,232 @@
+  },
+  "components/spotlight-section": {
+    "utrecht": {
+      "spotlight-section": {
+        "background-color": {
+          "$type": "color",
+          "$value": "{basis.color.text.interactive-1}"
+        },
+        "border-color": {
+          "$type": "color",
+          "$value": "transparent"
+        },
+        "color": {
+          "$type": "color",
+          "$value": "{basis.color.text.text-1}"
+        },
+        "info": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.info.bg-1}"
+          },
+          "border-color": {
+            "$type": "color",
+            "$value": "transparent"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.info.text-1}"
+          }
+        },
+        "error": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.error.bg-1}"
+          },
+          "border-color": {
+            "$type": "color",
+            "$value": "transparent"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.error.text-1}"
+          }
+        },
+        "ok": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.success.bg-1}"
+          },
+          "border-color": {
+            "$type": "color",
+            "$value": "transparent"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.success.text-1}"
+          }
+        },
+        "warning": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.warning.bg-1}"
+          },
+          "border-color": {
+            "$type": "color",
+            "$value": "transparent"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.warning.text-1}"
+          }
+        }
+      }
+    }
+  },
+  "components/select": {
+    "utrecht": {
+      "select": {
+        "background-color": {
+          "$type": "color",
+          "$value": "{basis.color.text.interactive-1}"
+        },
+        "border-radius": {
+          "$type": "dimension",
+          "$value": "{basis.border-radius.md}"
+        },
+        "focus": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.text.interactive-1}"
+          }
+        },
+        "invalid": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.text.interactive-1}"
+          }
+        }
+      }
+    }
+  },
+  "components/textarea": {
+    "utrecht": {
+      "textarea": {
+        "background-color": {
+          "$type": "color",
+          "$value": "{basis.color.text.interactive-1}"
+        },
+        "border-radius": {
+          "$type": "dimension",
+          "$value": "{basis.border-radius.md}"
+        },
```

Comment:
Ik stel voor dat we in het basis thema `basis.form-control.border-radius` maken, en daarnaar verwijzen in het basis thema in textarea/textbox, etc. Dan kunnen we 'm bij Purmerend in 1 keer instellen via die nieuwe token.